src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -666,7 +665,7 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
                 key="wizard-first-boot"
                 navItem={customStatusNavItem}
                 status={firstBootValidation.disabledNext ? 'error' : 'default'}
-                isHidden={!isFirstBootEnabled}
+                isHidden={!!process.env.IS_ON_PREMISE || isFedoraEnv}
```

Comment:
Based this on the tests in the end, assuming the first boot step shouldn't be visible in on-prem or Fedora service.